index.html

```diff
@@ -61,14 +61,12 @@ <h2>
         Dependencies
       </h2>
       <p>
-        The terms <dfn data-cite="HTML#the-link-element">link</dfn>, 
-        <dfn data-cite="HTML#insert-an-element-into-a-document">inserted into a document</dfn>, 
+        The terms <dfn data-cite="HTML#the-link-element">link</dfn>,
+        <dfn data-cite="HTML#insert-an-element-into-a-document">inserted into a document</dfn>,
         <dfn data-cite="HTML#resolve-a-url">resolve</dfn>, <dfn data-cite="HTML#url">url</dfn>, <dfn data-cite="HTML#attr-link-crossorigin">crossorigin</dfn>,
-        <dfn data-cite="HTML#origin-2">origin</dfn>, <dfn data-cite="HTML#delay-the-load-event">delay the load event</dfn>, <dfn data-cite="HTML#external-resource-link">external resource link</dfn>, <dfn data-cite="HTML#valid-media-query-list">valid media query list</dfn>, <dfn data-cite="HTML#queue-a-task">queue a task</dfn>, <dfn data-cite="HTML#fetch-and-process-the-linked-resource">fetch and process the linked resource</dfn> and <dfn data-cite="HTML#process-the-linked-resource">process the linked resource</dfn> are defined in [[!HTML]].
-      </p>
-      <p>
-        The terms <dfn data-cite="HTML52#match-the-environment">match the environment</dfn> 
-        are defined in [[!HTML52]].
+        <dfn data-cite="HTML#origin-2">origin</dfn>, <dfn data-cite="HTML#delay-the-load-event">delay the load event</dfn>, <dfn data-cite="HTML#external-resource-link">external resource link</dfn>, <dfn data-cite="HTML#valid-media-query-list">valid media query list</dfn>, <dfn data-cite="HTML#queue-a-task">queue a task</dfn>, <dfn data-cite="HTML#fetch-and-process-the-linked-resource">fetch and process the linked resource</dfn>,
+        <dfn data-cite="HTML#matches-the-environment">match the environment</dfn>,
+         and <dfn data-cite="HTML#process-the-linked-resource">process the linked resource</dfn> are defined in [[!HTML]].
```

Comment:
I don't think we need the '!' part before the "HTML" ref

Reply:
Fixed